### DISCUSSION TITLE
The trends doesn't display and return nothing from the API

### DISCUSSION THREAD

**Discussion by LeeYatSan** [mastodon/mastodon#27953] [Q&A]
Well, honestly, I am pretty new on Mastodon. I just set up my own instance with v3.0.1. Though I have turned on the Trending Now, the trends doesn't display on my public page. And when I try to use the API `https://mastodon.example/api/v1/trends` to get the JSON DATA ,it returns nothing but just a []. I search for the solution for a night but nothing useful. Waiting for you guys immediately reply!

**Comment by Gargron** [comment]
If it's a new instance is there any activity?

**Comment by LeeYatSan** [comment]
I am not sure what's your activity means. I have created 5 accounts and TooTed more than 60 statuses with more than 10 hashtags totally for test, but there is no any effect, which means the Trending Now doesn't display on the public page and no JSON data is returned by the API yet.

The following are some results after turning on the Trending Now, nothing changes :(
(Don't mind the statuses' content, just some meaningless text for test ~)  

<img width="898" alt="截屏2020-02-20上午11 19 30" src="https://user-images.githubusercontent.com/25663008/74897740-f660d480-53d2-11ea-8075-08362fea583e.png">

No JSON data is returned by `https://mastodon.example/api/v1/trends`
<img width="408" alt="截屏2020-02-20上午11 08 57" src="https://user-images.githubusercontent.com/25663008/74897349-c533d480-53d1-11ea-8f13-b649e076aea9.png">

<img width="954" alt="截屏2020-02-20上午11 06 02" src="https://user-images.githubusercontent.com/25663008/74897120-08da0e80-53d1-11ea-87b1-024fc6b3b3d6.png">

<img width="1271" alt="截屏2020-02-20上午11 25 07" src="https://user-images.githubusercontent.com/25663008/74898049-b9e1a880-53d3-11ea-84a8-667faceb0a80.png">

<img width="283" alt="截屏2020-02-20上午11 04 47" src="https://user-images.githubusercontent.com/25663008/74897118-07a8e180-53d1-11ea-9705-f55e7012467b.png">

<img width="677" alt="截屏2020-02-20上午11 05 16" src="https://user-images.githubusercontent.com/25663008/74897119-08417800-53d1-11ea-92f8-3a991cba3839.png">
